crates/sui-types/src/error.rs

```diff
@@ -450,6 +450,8 @@ pub enum VMMemoryLimitExceededSubStatusCode {
     NEW_ID_COUNT_LIMIT_EXCEEDED = 2,
     DELETED_ID_COUNT_LIMIT_EXCEEDED = 3,
     TRANSFER_ID_COUNT_LIMIT_EXCEEDED = 4,
+    OBJECT_RUNTIME_CACHE_LIMIT_EXCEEDED = 5,
+    OBJECT_RUNTIME_STORE_LIMIT_EXCEEDED = 6,
```

Comment:
Could you try adding a test to hit this? 
I _think_ you should be able to hit it without hitting the other one by creating a bunch of dynamic fields in a single transaction

Reply:
By `creating a bunch of dynamic fields in a single transaction`, you mean by calling `add`?
Thats what I did but could only hit the `OBJECT_RUNTIME_CACHE_LIMIT_EXCEEDED `. Unless there's another function you're speaking of?